palm/cli.py

```diff
@@ -31,6 +32,8 @@ def __init__(
     ) -> None:
         try:
             palm_config.validate_branch()
+        except NoRepositoryError:
+            palm_config.use_setup_plugins()
```

Comment:
bingo!

Reply:
I think we'd want to make some more changes to this to account for the new global plugin stuff. One thing we need to think through is whether all global plugins should be available outside of a configured git repo. Taking `palm-workflow` as an example, some commands in that plugin use the branch name pretty heavily, being outside fo a git repo would definitely cause problems in some places.